app-web/gatsby/utils/gatsbyRemark.js

```diff
@@ -99,6 +100,52 @@ const converter = (astType, path, markdownNode, parentQLnode, { getNode }) => {
     }
     return absolutePath;
   }
+  if (parentQLnode.internal.type === 'GithubRaw') {
+    // get all github raw nodes and curry
+    if (!converter.__githubRaw) {
+      converter.__githubRaw = getNodes().filter(n => n.internal.type === 'GithubRaw');
+    }
+
+    // normalize the path so that any paths that have no leading slash are assumed to be relative
+    const normalizedPath = normalizeFilePath(path);
+    const nodes = converter.__githubRaw;
+    let absolutePath;
+    // map out all nodes
+    if (isRelativePath(normalizedPath)) {
+      absolutePath = url.resolve(parentQLnode.html_url, normalizedPath);
+      // // we need to check if relative path would match a github path based on the collection
+      // // source locations map
+      const githubRawNodeThatMatches = nodes.find(n => {
+        return n.html_url.toLowerCase() === absolutePath.toLowerCase();
+      });
+
+      if (githubRawNodeThatMatches) {
+        absolutePath = `/${slugify(githubRawNodeThatMatches.___boundProperties.topics[0])}/${
```

Comment:
Maybe leave a comment to explain what is happening here incase people need to change in the future - besides that, looks great!